src/Components/customEditor/customEditor.js

```diff
@@ -27,38 +27,24 @@ export default class CustomEditor extends Component {
     }
 
     onFileUpload = (file) => {
-        var fileName = file.name;
-        var fileType = file.type;
+        const fileName = file.name;
+        const fileType = file.type;
         this.setState({fileName: fileName, fileType: fileType})
         var reader = new FileReader();
```

Comment:
reader is not getting changed right? can you check if we can also make it a `const`